mono/tests/sgen-domain-unload-2.cs

```diff
@@ -37,7 +40,33 @@ static void BackgroundNoise ()
 			AppDomain.Unload (ad);
 			Console.Write (".");
 			if (i > 0 && i % 20 == 0) Console.WriteLine ();
+			if (!testHasTimeLeft(startTime)) {
+				var finishTime = DateTime.UtcNow;
+				Console.WriteLine("stopping after {0}", finishTime-startTime);
```

Comment:
would be nice to print `i` here as well so we get an idea of how many iterations it managed.